kernel/GBEngine/kutil.cc

```diff
@@ -564,6 +564,9 @@ void cleanT (kStrategy strat)
         }
         else
         {
+          if (currRing->isLPring && strat->T[j].shift > 0) {
+            pNext(p) = NULL; // pNext(p) points to the unshifted tail, don't try to delete it here
```

Comment:
Bei den Polynomen in T wird nur das Leitmonom geshiftet, pNext zeigt auf pNext vom originalem Polynom. Damit es hier nur einmal gelöscht wird, wird es nur beim originalen Polynom gelöscht.